src/DataFixtures/FixtureBuilder.php

```diff
@@ -119,19 +119,27 @@ public static function createConference(array $description = []): Conference
             'endAt' => clone $date->modify('+ 1 days'),
             'cfpEndAt' => clone $date->modify('- 7 days'),
             'participations' => [],
+            'online' => false,
         ], $description);
 
         $conference = new Conference();
 
         ReflectionHelper::setProperty($conference, 'id', $description['id']);
         $conference->setName($description['name']);
-        $conference->setCity($description['city']);
-        $conference->setCountry($description['country']);
         $conference->setSiteUrl($description['siteUrl']);
         $conference->setCfpUrl($description['cfpUrl']);
         $conference->setStartAt($description['startAt']);
         $conference->setEndAt($description['endAt']);
         $conference->setCfpEndAt($description['cfpEndAt']);
+        $conference->setOnline($description['online']);
+
+        if (!$description['online']) {
+            $conference->setCity($description['city']);
+            $conference->setCountry($description['country']);
+        } else {
+            $conference->setCity('Online');
+            $conference->setCountry(null);
+        }
```

Comment:
Let's use the positive form before the negative on. I'll be easier to read:



```suggestion
        if ($description['online']) {
            $conference->setCity('Online');
            $conference->setCountry(null);
        } else {
            $conference->setCity($description['city']);
            $conference->setCountry($description['country']);
        }
```

BTW, the following line is useless, isn't?
```php
            $conference->setCountry(null);
```